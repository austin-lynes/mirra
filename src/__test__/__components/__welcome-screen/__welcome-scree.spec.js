import React from 'react';
import { shallow } from 'enzyme'
import WelcomeScreen from '../../../components/welcome-screen';

let welcomeScreen;

describe('components/welcome-screen', () => {

    beforeAll(() => {
        welcomeScreen = shallow(<WelcomeScreen />)
    })
    it('renders properly', () => {
        expect(welcomeScreen).toMatchSnapshot();
    })
})