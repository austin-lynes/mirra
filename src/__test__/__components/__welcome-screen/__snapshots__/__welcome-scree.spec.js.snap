// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/welcome-screen renders properly 1`] = `
<div
  className="welcome"
>
  <Modal
    close={[Function]}
    isLogin={true}
    visible={false}
  >
    <LoginForm />
  </Modal>
  <Modal
    close={[Function]}
    visible={false}
  >
    <RegisterForm
      setLoginVisible={[Function]}
    />
  </Modal>
  <h2
    className="welcome banner"
    data-testid="welcome-banner"
  >
    Welcome to Mirra!
  </h2>
  <div
    className="buttons"
  >
    <button
      data-testid="button"
      onClick={[Function]}
    >
      Login
    </button>
    <button
      data-testid="button"
      onClick={[Function]}
    >
      Register
    </button>
  </div>
</div>
`;
